src/Draco.Debugger.Tests/DebuggerTests.cs

```diff
@@ -0,0 +1,138 @@
+using System.Reflection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Draco.Debugger.Tests;
+
+public sealed class DebuggerTests
+{
+    private readonly ITestOutputHelper output;
+
+    public DebuggerTests(ITestOutputHelper output)
+    {
+        this.output = output;
+    }
+
+    private static async Task Timeout(Func<Task> action, int timeoutSecs = 10)
+    {
+        var task = action();
+        await task.WaitAsync(TimeSpan.FromSeconds(timeoutSecs));
+        if (!task.IsCompleted)
+        {
+            throw new TimeoutException();
+        }
+        await task; // await the task to throw any exceptions it might have thrown.
+    }
+
+    [SkippableFact]
+    public async Task SimpleProgramTerminate()
+    {
+        await Timeout(async () =>
```

Comment:
You could inline all of these to save on indent and make it a bit more fluent to read
```suggestion
    [SkippableFact]
    public async Task SimpleProgramTerminate() => await Timeout(async () =>
```